dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/ControlledVocabularyValueConverter.java

```diff
@@ -26,7 +26,7 @@ public Object getAsObject(FacesContext facesContext, UIComponent component, Stri
         if (submittedValue == null || submittedValue.equals("")) {
```

Comment:
Maybe it is because low java version. But isn't it possible to use `isEmpty()` here instead of `equals()`?
I also don't know if apache-commons is in dependencies list for this project, but if it is, then it is possible to use `StringUtils.isBlank()` which handles both emptiness of the string and it's nullability.

Reply:
Did not touch that part of the code, and I would like to not change code if not necessary. That's our current approach.